src/gc/impl/conservative/gc.d

```diff
@@ -383,9 +396,49 @@ struct GC
             return res;
     }
 
-    /**
-     *
-     */
+
+    void enable()
```

Comment:
That's fair. The only reason I did it was for consistency of order with the interface and the other implementation. I can change it back for reviewing.